httpobs/database/database.py

```diff
@@ -14,7 +14,7 @@
                           SCANNER_ABORT_SCAN_TIME)
 from httpobs.scanner import STATE_ABORTED, STATE_FAILED, STATE_FINISHED, STATE_PENDING, STATE_STARTING
 from httpobs.scanner.analyzer import NUM_TESTS
-from httpobs.scanner.grader import get_grade_for_score
+from httpobs.scanner.grader import get_grade_and_likelihood_indicator_for_score
```

Comment:
this function is not returning risk, but a likelihood_indicator. I named it after our last discussion on #infosec 
I can also name it get_grade_and_likelihood_for_score for example though ? @marumari